crt_portal/cts_forms/templates/landing.html

```diff
@@ -99,7 +99,7 @@ <h1 class="h1__display">{% trans "We uphold the civil rights of <em>all</em> peo
         <div class="tablet:grid-offset-2 tablet:grid-col-8">
           <img src="{% static "img/ic_phone-circle.svg" %}" alt="phone" class="icon">
           <p class="h3__display crt-landing--emergency_header">
-            {% trans 'If you are in danger, contact <a href="tel:911">9-1-1</a>' %}
+            {% trans 'If you are in danger, contact <a href="tel:911">911</a>' %}
```

Comment:
One thing I forgot to mention, we might want to change the corresponding translation string in the django po file. (In other words, the spanish translation also needs to be updated)